shopping-cart/index.html

```diff
@@ -13,7 +13,7 @@
     <header>
         <div class="gutter-right logo">
             <a href='../products/'>
-            <img src='./assets/dog-logo.png'>
+            <img src='/assets/dog-logo.png'>
```

Comment:
This works on my LiveServer, but I can't get it to work on the GitPages site for some reason.